Spring-Boot-News-Service/src/test/java/com/atomshermental/springbootnewsservice/NewsServiceTest.java

```diff
@@ -0,0 +1,47 @@
+package com.atomshermental.springbootnewsservice;
+
+import com.atomshermental.springbootnewsservice.model.News;
+import com.atomshermental.springbootnewsservice.model.NewsShablon;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.util.Assert;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class NewsServiceTest {
+
+    private NewsRepository newsRepository = new NewsRepository();
```

Comment:
You should create a mock of repository, because when you update the contents of repository, your test will fail